test/e2e_node/alignment/numa.go

```diff
@@ -0,0 +1,175 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package alignment
```

Comment:
Just to confirm, did you use `git mv` to move this from test/e2e_node/numa_alignment.go.  It would be nice to see the diff and preserve history.

Reply:
Initially yes but then it got lost in rebase. Thanks for pointing this out, I'll fix and push again.

Reply:
The last upload explicitely address this point. `git log --follow` works as expected.